values.yaml

```diff
@@ -260,7 +260,7 @@ ui:
 # Consul agent.
 syncCatalog:
   # True if you want to enable the catalog sync. "-" for default.
-  enabled: false
+  enabled: true
```

Comment:
This should stay `false` for the default helm chart.

Reply:
🤦‍♀️sorry, I thought I fixed this too.